bin/set_various.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 PROJECT_ID=$(jq -r '.TF_VAR_project_id' ../vars.json)
-DOCKER_REPO="eu.gcr.io"
+DOCKER_REPO="gcr.io"
```

Comment:
The gcr url (and in `vars_base` as well) is the eu component I was mentioning to look at for a possible EU component

Reply:
I made this change on my fork of this. I will test it and see if it works.